### PR TITLE
Use Jenkins git plugin

### PR DESCRIPTION
**Pull request by ackerleytng**

### CODE REVIEW

**Comment by ackerleytng** [comment]
Going to use the Jenkins git plugin to get commit and branch information! Also cleaning up as I go along

**Comment by demarey** [comment]
Hi Ackerley
Sorry for the very long delay! Thank you for your contribution.
We started to review it and will update it so that it can be integrated into Pharo 10 branch.

File: Makefile [hunk 1]
```diff
@@ -1,149 +1,206 @@
...
-	$(bootstrap) ./bootstrap/scripts/prepare_image.st --save --quit
-
-Pharo.image:
-	wget https://github.com/guillep/PharoBootstrap/releases/download/v1.4.1/bootstrapImage.zip
-	unzip bootstrapImage.zip
-
-pharo-vm:
-	wget -O - get.pharo.org/vm61 | bash
-	
+.PHONY: bootstrap_tools target_vm clean bootstrap
```
Comment (demarey) [inline Makefile:1]: nice! I learned something on Makefiles

**Comment by demarey** [comment]
@ackerleytng The title of this PR does not reflect changes done or I miss something. What do you mean by "use Jenkins git plugin"?

**Comment by ackerleytng** [comment]
Oh man, sorry about the poor title, and also, didn't realize this was such a huge PR. It was quite long ago and I can't really remember all the changes I made around this.

I made some changes around the build and bootstrap process... and I understand it's quite a core change for Pharo. Probably quite many things have changed since then. I'm ok if you want to just close this and not merge it.

**Comment by demarey** [comment]
Let's integrate all the documentation effort.
Makefile needs to be reviewed but the old version is still there.
I will open an issue for the review.
Thank you for the contribution.

**Pull request merged**